airflow/www/security.py

```diff
@@ -200,6 +200,12 @@ def __init__(self, appbuilder):
             view.datamodel = CustomSQLAInterface(view.datamodel.obj)
         self.perms = None
 
+    def _is_subdag(self, dag_id):
+        if '.' in dag_id:
+            dm = self.get_session.query(DagModel.is_subdag).filter(DagModel.dag_id == dag_id).first()
+            return dm.is_subdag if dm else False
```

Comment:
```suggestion
            return self.get_session.query(DagModel.is_subdag).filter(DagModel.dag_id == dag_id).scalar()
```

This should work as well. It returns `None` if the DAG does not exist, which is fine for its usages.